components/tikv_kv/src/stats.rs

```diff
@@ -251,3 +251,23 @@ impl StatisticsSummary {
         self.count += 1;
     }
 }
+
+/// Latency indicators for multi-execution-stages.
+///
+/// The detailed meaning of the indicators is as follows:
+///
+/// ```text
+/// ------> Begin ------> Scheduled ------> SnapshotReceived ------> Finished ------>
+/// |----- schedule_wait_time -----|
+///                                |-- snapshot_wait_time --|
+/// |------------------- wait_wall_time --------------------|
+///                                                         |-- process_wall_time --|
+/// |------------------------------ kv_read_wall_time ------------------------------|
```

Comment:
This model might not fully work for all requests. I guess transactions have different stages. This could be a blocking stone to generalize it. It might also not work for batch get commands (which merges multiple get requests to be one).

Reply:
I recorded this note in roadmap issue: https://github.com/tikv/tikv/issues/11525